script/foreman-admin

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env ruby
+# -*- encoding: utf-8 -*-
+
+require 'rubygems'
+require 'clamp'
```

Comment:
Sure, but one can still install foreman without hammer and clamp will not get pulled in as dependency then... At least clamp (for RH and friends in the non-SCL variant) has to get added as dependency in the packaging.